### ISSUE TITLE
Compile Error gmic-qt

### ISSUE THREAD

**Issue by ClaesGitHub** [c-koi/gmic-qt#71]
Hello/Salut!
Manjaro KDE 18.0.4
I am trying to compile gmic-qt according to directions given on the GitHub page above.
It worked just fine yesterday, but today I get a compile error:
```
./gmic/src/gmic.cpp: In static member function ‘static const cimg_library::CImg<char>& gmic::decompress_stdlib()’:
../gmic/src/gmic.cpp:2474:59: error: ‘data_gmic_stdlib’ was not declared in this scope
       CImgList<char>::get_unserialize(CImg<unsigned char>(data_gmic_stdlib,1,size_data_gmic_stdlib,1,1,true))[0].
                                                           ^~~~~~~~~~~~~~~~
../gmic/src/gmic.cpp:2474:78: error: ‘size_data_gmic_stdlib’ was not declared in this scope
 ist<char>::get_unserialize(CImg<unsigned char>(data_gmic_stdlib,1,size_data_gmic_stdlib,1,1,true))[0].

```
Cordially,
Claes in Lund, Sweden

**Comment by dtschump** [comment]
This is a variable defined in file `gmic_stdlib.h`. As this file is automatically generated from the file `gmic_stdlib.gmic`, it is not included in the git repo, but downloaded from the G'MIC web server https://gmic.eu/gmic_stdlib.h (which is done by the Makefile).
Maybe something went wrong when trying to download it, so you may try to delete it and try again (or copy paste it from the URL I've mentioned).

**Comment by ClaesGitHub** [comment]
Salut & thanks for swift reply.

I removed all files and downloaded them again, according to instructions here: 
https://gmic.eu/download.shtml, that is:
        $ git clone --depth=1 https://framagit.org/dtschump/gmic.git
        $ git clone --depth=1 https://github.com/c-koi/gmic-qt.git
        $ git clone --depth=1 https://github.com/c-koi/zart.git
Followed by
        $ cd gmic/src
        $ make gimp -j16
Error, as above.

The Makefile _did_ create a file named gmic_stdlib.h in .../gmic/src, but it is _empty_.
I fetched a fresh gmic_stdlib.h according to your instructions above, and placed it in ..../gmic/src.
issued a new _make gimp -j16_  and now it compiles just fine.

What bothers me is that compilation worked just fine _before today_. What happened during this morning's git pull?
